packages/cozy-search/src/components/ResultMenu/ResultMenuItem.jsx

```diff
@@ -46,8 +48,24 @@ const ResultMenuItem = ({
     secondaryText
   )
 
+  useEffect(() => {
+    if (selected) {
+      itemRef?.current?.scrollIntoView({
+        behavior: 'smooth',
```

Comment:
nit: if the values are those by default, maybe we can remove these key to simplify